### PR TITLE
added username swapper

### PR DESCRIPTION
**Pull request by AshIgnYeo**
checks for username if not displays email

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <h1>Pages#home</h1>
 <p>
-  <%= current_user.email if user_signed_in? %>
+  <%= current_user.username || current_user.email if user_signed_in? %>
```
Comment (MysteryGenius) [inline app/views/pages/home.html.erb]: Change to ternary operator tks

**Review by MysteryGenius — CHANGES_REQUESTED** [review]
Yeah looks good but changes to logics

**Review by MysteryGenius — APPROVED** [review]
Looks good! Approved

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (AshIgnYeo) [reply app/views/pages/home.html.erb]: ok boss. done. please tell me if can merge not

**Pull request merged**